types/react-native-simple-animations/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "private": true,
+    "name": "@types/react-native-simple-animations",
+    "version": "0.3.9999",
+    "projects": [
+        "https://www.npmjs.com/package/react-native-simple-animations"
+    ],
+    "dependencies": {
+        "@types/react": "*",
+        "react-native": "*"
+    },
+    "devDependencies": {
+        "@types/react-native-simple-animations": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Joey Schroeder",
+            "githubUsername": "joeyschroeder"
+        },
```

Comment:
Please don't add people unless they explicitly ask to be added. I would doubt they want to be here.